internal/dataplane/kong_client.go

```diff
@@ -617,3 +622,30 @@ func (c *KongClient) recordResourceFailureEvents(resourceFailures []failures.Res
 		}
 	}
 }
+
+// recordApplyConfigurationEvents records event attached to KIC pod after KIC applied Kong configuration.
+func (c *KongClient) recordApplyConfigurationEvents(err error, rootURL string) {
+	eventType := corev1.EventTypeNormal
+	reason := "KongConfigApplySucceeded"
+	message := fmt.Sprintf("successfully applied Kong configuration to %s", rootURL)
+
+	if err != nil {
+		eventType = corev1.EventTypeWarning
+		reason = "KongConfigApplyFailed"
```

Comment:
Isn't it duplicating the already existing `KongConfigurationApplyFailed` event reason?

Reply:
Done